spec/shell-session-spec.coffee

```diff
@@ -1,12 +1,14 @@
 describe "Shell session grammar", ->
   grammar = null
+  grammar_session = null
 
   beforeEach ->
     waitsForPromise ->
       atom.packages.activatePackage("language-shellscript")
 
     runs ->
-      grammar = atom.grammars.grammarForScopeName("text.shell-session")
+      grammar_session = atom.grammars.grammarForScopeName("text.shell-session")
```

Comment:
I was planning on fixing the specs in a :art: PR later since Shellscript's specs look pretty different compared to others'.